### DISCUSSION TITLE
de-google: best way to transfer mail from gmail (for business)

### DISCUSSION THREAD

**Discussion by Waldorf3** [docker-mailserver/docker-mailserver#4290] [Q&A] [answered]
What is the best way to transfer mail from gmail?   I was thinking perhaps fetchmail, but maybe that needs directly access to the mbox.  There is an imapsync app, but as far as I can see it's a 150€ license.   Could of course install a desktop client and sync everything down from gmail and then up to mailserver, but it seems a bit long form.   Any better ideas?

**Comment by casperklein** [comment, marked as answer]
You can try https://github.com/imapsync/imapsync

**Reply by Waldorf3** [reply]
Thanks! I had a look at it already but thought it was commercial software with a 150€ license fee. Now I checked more carefully I can see that's optional.  In fact I found there's a docker version with an interface making the process very easy: https://hub.docker.com/r/gilleslamiral/imapsync
